servant/test/Servant/Utils/LinksSpec.hs

```diff
@@ -26,7 +26,7 @@ type TestApi =
   :<|> "put" :> Put '[JSON] ()
   :<|> "post" :> ReqBody '[JSON] 'True :> Post '[JSON] ()
   :<|> "delete" :> Header "ponies" String :> Delete '[JSON] ()
-  :<|> "raw" :> Raw
+  :<|> "raw" :> Raw () IO
```

Comment:
Can all these `IO`s be just `m`? Semantically, I would expect it to be `EitherT ServantErr IO`, but I guess when `enter` isn't used it doesn't matter?